### PR TITLE
Integrate upcoming Community Solid Server release into the website

### PR DESCRIPTION
**Pull request by RubenVerborgh**
This PR inserts documentation on how to install Community Solid Server v1.0.

### CODE REVIEW

**Comment by csarven** [comment]
Remind me.. is there a reference to where consensus was reached on this change/recommendation?

**Comment by jeff-zucker** [comment]
May I ask about the intended transition process?  I think it would *really* help implementers if there could be a period when CSS runs on an alternate port for testing before it replaces NSS on the main port.  It would also be really helpful to have a rough timeline of when the testing server would be available and when the main port is targeted for switching to CSS.  I also have questions about solidcommunity.net as an IDP - will the same registration processes be available?

**Comment by RubenVerborgh** [comment]
> Remind me.. is there a reference to where consensus was reached on this change/recommendation?

There is none; just trying to be helpful with getting the tutorial up to date.
Is there consensus needed for this beyond a review on this PR?

> May I ask about the intended transition process?

Absolutely, we need one. And that's going to be a longer conversation.

This is just ensuring that new developers start out with the most spec-compatible Node.js implementation, to avoid creating new apps with old authentication methods, for instance.

>  I think it would _really_ help implementers if there could be a period when CSS runs on an alternate port for testing before it replaces NSS on the main port.

> I also have questions about solidcommunity.net as an IDP

I'm not sure if we're on the same page here; solidcommunity.net is _not_ being replaced at this point in time.

This PR upgrades the docs for new developers to recommend CSS rather than NSS.

**Comment by jeff-zucker** [comment]
Yes, I see now that prepping the onboarding materials is separate from transition of solidcommunity.net.  Is there an issue about the transition on solidcommunity.net?  If not, I'll create one.  

I agree that it's a good idea to have the onboarding materials recommend CSS to new developers.What I would also like to see in the onboarding materials is a section aimed at existing developers who have aimed their  apps at NSS.  What should we expect in terms of differences?  What things might we have wrongly gotten used to in NSS that won't work the same when spec compliant.

**Comment by RubenVerborgh** [comment]
> Is there an issue about the transition on solidcommunity.net?

I'm not sure if there is an issue; but I know that @justinwb is following up on that.

> What I would also like to see in the onboarding materials is a section aimed at existing developers who have aimed their apps at NSS.

Okay if we track that in https://github.com/solid/solidproject.org/issues/607?

**Comment by csarven** [comment]
>There is none; just trying to be helpful with getting the tutorial up to date.

Ack. Replacing NSS with CSS is a significant change. Ditto any implications on existing applications. (This is besides the point on moving things forward..)

>Is there consensus needed for this beyond a review on this PR?

Perhaps the PR is sufficient but given its significance I think it'd be appropriate to raise this PR in the next Solid Team meeting, and make sure there are no objections, and concerns are acknowledged. It'd be good to mention the PR in solid/* chats etc.

**Comment by RubenVerborgh** [comment]
> Replacing NSS with CSS is a significant change.

It will be; but that's not the scope of this PR.

This PR adds new CSS documentation and removes old NSS documentation.
We deliberately do not make any suggestions about NSS being replaced, because that is not happening.

> It'd be good to mention the PR in solid/* chats etc.

Indeed; tracking at:
- https://gitter.im/solid/team?at=610aa4baf31bc0605a4537f7
- https://gitter.im/solid/chat?at=610aa5a13eec6d41d13b5ad4

> raise this PR in the next Solid Team meeting

Unless this PR attracts significant feedback, will raise on 2021-08-11.

**Comment by timbl** [comment]
"This is just ensuring that new developers start out with the most spec-compatible Node.js implementation, to avoid creating new apps with old authentication methods, for instance." -- well, NSS supports new auth methods as well as the old ones.  The solid-ui stack needs to be upgraded.   This is in process.  Does the databrowser in general work with CSS? I suggest the solidproject.org mention both NSS and CSS for a while, don't just eliminate NSS.

**Comment by RubenVerborgh** [comment]
> Does the databrowser in general work with CSS?

It does not; because it using old authentication protocols. And that's exactly what we want to avoid: new people starting out with old things.

> I suggest the solidproject.org mention both NSS and CSS for a while, don't just eliminate NSS.

Point taken, I'll just change this PR to keep including NSS, and then there's nothing controversial.

**Comment by ewingson** [comment]
read attentively. as an contributor to https://solidproject.org/self-hosting/nss and maintainer of https://solidweb.org I think it's best to drive simultanously (NSS and CSS) for a while. perhaps onboarding for new people to CSS makes sense. as I can say for today, I will follow the switch of solidcommunity.net when it happens and do my best to admin the part I am responsible for as good as my knowledge is. thumbs up for discussion and building consensus.

File: _posts/developers/apps/tools/2019-01-01-00_overview.md
Comment (csarven) [inline _posts/developers/apps/tools/2019-01-01-00_overview.md:53]: I'm not up to date with this.. I suppose it was put to sleep. Is this repo to be archived or still in use?

File: pages/faqs.md
Comment (csarven) [inline pages/faqs.md:80]:
```suggestion
The Solid standard is open as are some of its implementations such as the <a href="https://github.com/solid/community-server">Community Solid Server</a> meaning that anyone can self-host their own Identity and Pod or the Identity and Pod of a group of users. 
```

**Review by csarven — APPROVED** [review]
Useful to introduce CSS...

File: _posts/developers/pod-server/2019-01-02-00_apache-config.md
Comment (kay-kim) [inline _posts/developers/pod-server/2019-01-02-00_apache-config.md:82]: Should we mention that the example config uses `example.org` and as such, people should substitute their server?

File: _posts/developers/pod-server/2019-01-02-00_apache-config.md
Comment (kay-kim) [inline _posts/developers/pod-server/2019-01-02-00_apache-config.md:13]: I think the , is unnecessary(?) since its "A reverse proxy allows you to run .. and let..."?

File: _posts/developers/pod-server/2020-08-12-00_nginx-config.md
Comment (kay-kim) [inline _posts/developers/pod-server/2020-08-12-00_nginx-config.md:14]: ditto on the comma

File: _posts/developers/pod-server/2019-01-01-00_overview.md
Comment (kay-kim) [inline _posts/developers/pod-server/2019-01-01-00_overview.md:19]: Since we're listing more than a couple and as the list grows, don't want to keep a count:

```suggestion
The following  lists some available **open-source servers**:
```

or

```suggestion
The following **open-source servers** are available:
```

**Review by jeff-zucker — APPROVED** [review]

File: _posts/developers/pod-server/2019-01-01-00_overview.md
Comment (kay-kim) [inline _posts/developers/pod-server/2019-01-01-00_overview.md:102]:
```suggestion
You can expose the server directly either by running it on a public port
or by placing it behind a _reverse proxy_.
```

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md
Comment (kay-kim) [inline _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:21]:
```suggestion
Various [Solid Pod providers]((/users/get-a-pod#get-a-pod-from-a-pod-provider) exist.
```

File: _posts/developers/apps/tools/2019-01-01-00_overview.md
Comment (RubenVerborgh) [reply _posts/developers/apps/tools/2019-01-01-00_overview.md:53]: It's directly included in NSS and has no use beyond NSS, so its separate listing is an accident.

File: _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md
Comment (kay-kim) [inline _posts/developers/apps/inrupt-tutorial/2021-08-03-00_getting-started.md:36]:
```suggestion
you're ready to start [developing your own app](/developers/tutorials/first-app).
```

**Comment by RubenVerborgh** [comment]
Thanks @ewingson @csarven @jeff-zucker @timbl, merging the addition.

**Comment by RubenVerborgh** [comment]
Thanks @kay-kim, will still take your suggestions into account before merging.

**Pull request merged**
